flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/sink/HBaseSinkFunction.java

```diff
@@ -76,6 +80,7 @@
     private transient ScheduledExecutorService executor;
     private transient ScheduledFuture scheduledFuture;
     private transient AtomicLong numPendingRequests;
+    private Map<ByteBuffer, Mutation> mutationMap = new HashMap<>();
```

Comment:
This field could be `final` as well. Can we move it under `mutationConverter` and init it inside the ctor to be consistent with the field initialization?